components/eventing-controller/pkg/env/nats_config.go

```diff
@@ -25,8 +25,14 @@ type NatsConfig struct {
 	IdleConnTimeout     time.Duration `envconfig:"IDLE_CONN_TIMEOUT" default:"10s"`
 
 	// JetStream-specific configs
-	JSStreamName            string `envconfig:"JS_STREAM_NAME" default:"kyma"`
-	JSStreamStorageType     string `envconfig:"JS_STREAM_STORAGE_TYPE" default:"memory"`
+	// Name of the JetStream stream where all events are stored.
+	JSStreamName string `envconfig:"JS_STREAM_NAME" default:"kyma"`
+	// Storage type of the stream, memory or file.
+	JSStreamStorageType string `envconfig:"JS_STREAM_STORAGE_TYPE" default:"memory"`
+	// Retention policy specifies when should events be deleted from the stream.
```

Comment:
```suggestion
	// Retention policy specifies which events to delete from the stream.
```

Reply:
It's not which, it's when.

Reply:
Alrighty then ¯\_(ツ)_/¯ 

```suggestion
	// Retention policy specifies when to delete events from the stream.
```